charts/tidb-cluster/values.yaml

```diff
@@ -192,6 +192,15 @@ tidb:
         cpu: 20m
         memory: 5Mi
 
+  # tidb plugin configuration
+  plugin:
+    # enable plugin or not
+    enable: false
+    # the start argument to specify the folder containing
+    directory: /plugins
+    # the start argument to specify the plugin id (name "-" version) that needs to be loaded, e.g. 'conn_limit-1'.
+    list: whitelist-1
```

Comment:
If there're multiple plugins, is list a comma separated string? I found a example, but not sure It works or not.https://github.com/helm/charts/blob/master/stable/opa/templates/deployment.yaml#L96

Reply:
yes!
From the design document.
https://github.com/pingcap/tidb/blob/master/docs/design/2018-12-10-plugin-framework.md#Management

and tidb uage document 
https://github.com/pingcap/tidb/blob/11d7199fcfbb66b1205ffa00f770f90f7b1ee9c1/tidb-server/main.go#L114

and the code that load plugins
https://github.com/pingcap/tidb/blob/11d7199fcfbb66b1205ffa00f770f90f7b1ee9c1/session/session.go#L1305
```go
	if len(cfg.Plugin.Load) > 0 {
		err := plugin.Load(context.Background(), plugin.Config{
			Plugins:        strings.Split(cfg.Plugin.Load, ","),
			PluginDir:      cfg.Plugin.Dir,
			GlobalSysVar:   &variable.SysVars,
			PluginVarNames: &variable.PluginVarNames,
		})
		if err != nil {
			return nil, err
		}
	}
```

Reply:
the plugin.list is changed to array